src/main/java/com/iexec/sms/config/SecretsConfig.java

```diff
@@ -65,56 +69,64 @@ void shutdown() throws InterruptedException {
 
     @Bean
     MeasuredSecretService web2MeasuredSecretService(Web2SecretRepository web2SecretRepository,
-                                                    @Value("${metrics.storage.refresh-interval}") int storedSecretsCountPeriod) {
+                                                    @Value("${metrics.storage.refresh-interval}") int storedSecretsCountPeriod,
+                                                    CacheSecretService<Web2SecretHeader> web2CacheSecretService) {
```

Comment:
I tend to order parameters as follows:
1. Services
2. Configuration values

It seems more natural to me to read parameters in that way. Do you agree?